lib/parse.js

```diff
@@ -124,7 +124,7 @@ function parseFile (file, options, $refs) {
       .then(onParsed, onError);
 
     function onParsed (parser) {
-      if ((options.continueOnError || !parser.plugin.allowEmpty) && isEmpty(parser.result)) {
+      if (!options.continueOnError && !parser.plugin.allowEmpty && isEmpty(parser.result)) {
```

Comment:
This is close, but not quite correct.
We should still throw when `continueOnError`  true, so the condition could look as follows
```suggestion
      if (!parser.plugin.allowEmpty && isEmpty(parser.result)) {
```
You may need to pass `allowEmpty` in exporter service

Reply:
`allowEmpty` default to true && I fixed this `continueOnError`.